assets/components/atoms/icon/icon-list.twig

```diff
@@ -0,0 +1,10 @@
+<table class="table bg-gray-100">
+  <tbody>
+    {% for icon in icons %}
```

Comment:
It will not magically looping over all the icons contain in the dedicated directory... Please test more before delivering something...

See example [datas](https://github.com/antistatique/IMD-TL-Styleguide/blob/master/assets/config/data.json#L2)

Reply:
Ideed. It did work perfectly for my social icons, so I didn't bother ^^ Fixed now, and I see how It's supposed to look now :D